pages/model/[uuid]/new-version.tsx

```diff
@@ -142,7 +143,8 @@ function Upload() {
 
     form.append('code', data.files.code)
     form.append('binary', data.files.binary)
-
+    form.append('seldonVersion', data.files.seldonVersion)
```

Comment:
Seldon version is _not_ a file and so must not be added as an extra field, it should remain in the model card like every other item.